webcomponents/src/app/components/gn-results-list/gn-results-list.sample.html

```diff
@@ -8,19 +8,21 @@
   </head>
   <body>
     <script src="gn-wc.js"></script>
-    <div>
-      <button id="changeSizeBtn">Change size</button>
-    </div>
     <gn-results-list
       api-url="https://apps.titellus.net/geonetwork/srv/api"
       size="10"
       layout="TITLE"
-      filters="+tag.default:Soil"
+      filter="GRID"
```

Comment:
what is it ?

Reply:
https://github.com/geonetwork/geonetwork-ui/blob/master/webcomponents/gn-results-list/src/app/gn-results-list.component.ts#L24 is `filter` not `filters` but then the example can be whatever. But the attribute was wrong.